source/configure/calls-deployment.rst

```diff
@@ -338,10 +342,35 @@ If you wish to host many calls or calls with a large number of participants, tak
   # Allow to allocate more memory as needed for more control messages that need to be sent for each socket connected
   net.core.optmem_max = 16777216
 
+The rtcd service
+----------------
+
+.. include:: ./calls-rtcd-ent-only.rst
+  :start-after: :nosearch:
+
+The Calls plugin has a built-in `Selective Forwarding Unit (SFU) <https://bloggeek.me/webrtcglossary/sfu/>`__ to route audio and screensharing data. But this SFU functionality can be split from the Calls plugin and hosted on its own, e.g., on its own node in a `kubernetes deployment <#kubernetes-deployments>`__, or its own dedicated server `alongside Mattermost instances <#modes-of-operation>`__ in a cloud service or self-hosted.
+
+Reasons to use the ``rtcd`` service
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+It helps to understand when and why your organization would want to use ``rtcd``.
+
+First of all, ``rtcd`` is a standalone service, which adds operational complexity. For most small instances of Mattermost there is no need to use ``rtcd``. Instead, use the Calls plugin until performance or reliability becomes an issue.
+
+- **Performance and stability of the main Mattermost server(s).** Letting the Calls plugin run the SFU means that calls traffic is being processed by the server that's also processing the rest of the Mattermost data (Channels, Boards, Playbooks, etc). If the Calls traffic spikes, it can affect the responsiveness of the rest of the Mattermost services. Using an ``rtcd`` service isolates the calls traffic processing to those ``rtcd`` instances.
+- **Performance, scalability, and stability of the Calls product.** If Calls traffic spikes, or more overall capacity is needed, ``rtcd`` servers can be added to balance the load. As an added benefit, if the Mattermost traffic spikes, or if a Mattermost instance needs to be restarted, those people in a current call will not be affected - current calls won't be dropped. 
+
+Some caveats apply here, for example emoji reactions will not be transmitted while the main Mattermost server is down. But the call itself will be resilient to main server restarts.
```

Comment:
That would be the case for any websocket event (muting/unmuting, even signaling at this point). Maybe let's add a generic "websocket events" since it covers more than purely reactions.